web/src/containers/ActivityDetailStandardsAndConditions.js

```diff
@@ -0,0 +1,72 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { STANDARDS } from '../util';
+
+import { updateActivity as updateActivityAction } from '../actions/activities';
+import Collapsible from '../components/Collapsible';
+import { Textarea } from '../components/Inputs2';
+
+class ActivityDetailStandardsAndConditions extends Component {
+  handleChange = key => e => {
+    const { value } = e.target;
+    const { activity, updateActivity } = this.props;
+
+    const updates = { standardsAndConditions: { [key]: value } };
+    updateActivity(activity.id, updates);
+  };
+
+  render() {
+    const { activity } = this.props;
+
+    return (
+      <Collapsible title="Standards & Conditions">
+        <p>Tell us how you’ll meet the Medicaid standards and conditions.</p>
+        {STANDARDS.map(std => {
+          const inputId = `a-${activity.id}-standards-${std.id}`;
+          return (
+            <div key={std.id}>
+              <h3>{std.title}</h3>
+              <p>
+                Sed ut perspiciatis unde omnis iste natus error sit voluptatem
+                accusantium doloremque laudantium, totam rem aperiam, eaque ipsa
+                quae ab illo inventore veritatis et quasi architecto...
+              </p>
+              <div>
+                <label htmlFor={inputId}>
```

Comment:
no need for this label (and wrapping div) anymore -- our `Textarea` component should now include both of those